src/TableFunctions/ITableFunctionFileLike.cpp

```diff
@@ -25,24 +25,6 @@ namespace ErrorCodes
     extern const int BAD_ARGUMENTS;
 }
 
-namespace
-{
-    void checkIfFormatSupportsAutoStructure(const String & name, const String & format)
-    {
-        if (name == "file" && format == "Distributed")
```

Comment:
What is Distributed format?

Reply:
You can look here: https://github.com/ClickHouse/ClickHouse/pull/8535